### PR TITLE
Add tool for removing variables from netcdf files

### PR DESCRIPTION
**Pull request by JamieJQuinn**
This PR adds a small tool which can strip a variable from a netcdf file. This is useful for datasets where there is more than one variable, which can be an issue for the loaders (see #59).

### CODE REVIEW

**Review by alessandrofelder — COMMENTED** [review]
Is it worth adding a very simple example in the README where the other scripts are documented?

**Comment by Devaraj-G** [comment]
@JamieJQuinn 
Does this default to `Band 1` if multiple bands are present in the input file?
Or do I have to specify the band?

**Comment by JamieJQuinn** [comment]
> @JamieJQuinn
> Does this default to `Band 1` if multiple bands are present in the input file?
> Or do I have to specify the band?

@Devaraj-G 
This tool just delete variables, it doesn't touch any bands at all. 

In remove-restore, if there are multiple bands I think it will fail, but AFAIK the bands are usually only important for satellite data (i.e. different bands for different wavelengths). If we come up against that kind of problem in a specific file we can deal with it.

**Comment by Devaraj-G** [comment]
> > @JamieJQuinn
> > Does this default to `Band 1` if multiple bands are present in the input file?
> > Or do I have to specify the band?
> 
> @Devaraj-G
> This tool just delete variables, it doesn't touch any bands at all.
> 
> In remove-restore, if there are multiple bands I think it will fail, but AFAIK the bands are usually only important for satellite data (i.e. different bands for different wavelengths). If we come up against that kind of problem in a specific file we can deal with it.

So, the problematic dataset in [#59](https://github.com/UCL/pyCascadia/issues/59)  need be still pre-processed? Or, this PR solves it.

**Comment by JamieJQuinn** [comment]
> So, the problematic dataset in #59 need be still pre-processed? Or, this PR solves it.

Yes, it will still have to be pre-processed. I can't think of an easy, general way to pick the correct variable from a dataset containing multiple variables. Are there many files like that?

**Comment by Devaraj-G** [comment]
> > So, the problematic dataset in #59 need be still pre-processed? Or, this PR solves it.
> 
> Yes, it will still have to be pre-processed. I can't think of an easy, general way to pick the correct variable from a dataset containing multiple variables. Are there many files like that?

Maybe less than 10 `.nc` files. There are [some](https://coast.noaa.gov/htdata/raster2/elevation/NCEI_ninth_Topobathy_2014_8483/wash_bellingham/) NOAA `TIFFs` too. Haven't checked if they create problems.

**Comment by JamieJQuinn** [comment]
> Maybe less than 10 .nc files. There are some NOAA TIFFs too. Haven't checked if they create problems.

OK. LMK if more files like strait_of_juan... turn up and we can maybe figure out a better way than just pre-processing each one.

File: tests/test_utility.py [hunk 1]
```diff
@@ -21,3 +22,18 @@ def test_read_fnames():
...
+
+    # Ensure ds initially contains the variable
+    assert 'crs' in ds
+
+    # Ensure deleting works
+    delete_variable(ds, 'crs')
+    assert 'crs' not in ds
+
+    # Ensure deleting a missing variable throws
+    with pytest.raises(Exception):
```
Comment (alessandrofelder) [inline tests/test_utility.py:38]: Tiny suggestion
```suggestion
    with pytest.raises(ValueError, match=f"Could not find crs in dataset"):
```

**Review by alessandrofelder — COMMENTED** [review]
Great job, just a tiny suggestion, then I'll approve :)

**Review by alessandrofelder — APPROVED** [review]
👍

**Pull request merged**